<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileInfoDisplay</class>
 <widget class="QWidget" name="FileInfoDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>70</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>70</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>70</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="icon">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>65</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>65</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="fileName">
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="omniLayout">
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>5</height>
      </size>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
